hyperspy/signal.py

```diff
@@ -6414,6 +6458,58 @@ def _check_signal_mask(self, mask):
             raise ValueError("The shape of signal mask array must match "
                              "`signal_shape`.")
 
+    def to_gpu(self):
```

Comment:
I'm wondering whether it might not be better to use CUDA terminology in these functions, e.g. `to_device` and `to_host`. Reasons being only CUDA enabled GPUs will work, and `to_cpu` is not entirely correct as a name with regards to what is happening.

Reply:
> I'm wondering whether it might not be better to use CUDA terminology in these functions, e.g. `to_device` and `to_host`.

Yes, I agree, I went for `to_cpu` and `to_gpu`, because the meaning is straightforward for anyone - although this assume that the user understand that the data is in memory somewhere - but this is better to use the device/host terminology: it is simple to explain in the user guide and it will be consistent with other documentation.

> `to_cpu` is not entirely correct as a name with regards to what is happening.

I don't understand, why using `to_device` or `to_host` would be better?